### ISSUE TITLE
Manga Bilgini has third party software protection

### ISSUE THREAD

**Issue by kyoyacchi** [tachiyomiorg/tachiyomi-extensions#12879]
### Source information and language

Manga Bilgini 1.3.24 (Turkish)

### Steps to reproduce

1. Try to open a manga in Manga Bilgini extension.
2. You'll see a chapter named "Never Gonna Give You Up"
3. it has a rickroll gif and information message.

### Expected behavior

It should work normally.

### Actual behavior

It says "You are using a third party software, third party softwares are blocked by site against series gets stealed."

![Screenshot_20220806-183709](https://user-images.githubusercontent.com/63583961/183255868-77e4e41c-829a-428a-9827-5beac1a9f771.jpg)

### Tachiyomi version

TachiyomiJ2K 1.5.6

### Android version

Android 11

### Other details

https://mangabilgini.com/

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.5](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.
